From 93c2f9bd8b058b28016e6db2421e5b38eac0606c Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Thu, 17 Dec 2015 15:06:38 +0100
Subject: [PATCH] Revert to old pre-1.0 meta directory

Unionfs changed its meta directory from .unionfs to .unionfs-fuse with the
unionfs -> unionfs-fuse rename.  The rename later got reverted everywhere
but the meta directory, so now unionfs doesn't find the whiteout files from
older releases.

Revert back to the pre-1.0 behaviour to fix this.

Committed upstream:
https://github.com/rpodgorny/unionfs-fuse/commit/93c2f9bd8b058b28016e6db2421e5b38eac0606c

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 src/unionfs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/unionfs.h b/src/unionfs.h
index c2b5ee3..74b824f 100644
--- a/src/unionfs.h
+++ b/src/unionfs.h
@@ -10,7 +10,7 @@
 #define PATHLEN_MAX 1024
 #define HIDETAG "_HIDDEN~"
 
-#define METANAME ".unionfs-fuse"
+#define METANAME ".unionfs"
 #define METADIR (METANAME  "/") // string concetanation!
 
 // fuse meta files, we might want to hide those
-- 
2.1.4

